cmd/agent-operator/DEVELOPERS.md

```diff
@@ -41,6 +41,12 @@ go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.2
 go install github.com/ahmetb/gen-crd-api-reference-docs@v0.3.1-0.20220618162802-424739b250f5
 ```
 
+Use the following to run the script in a container:
+
+```
+docker build -t grafana/agent-build-image:0.18.0 ./build-image
```

Comment:
This version number is too specific, maybe we could just use `latests` as a tag.